runtime/tools/spec-builder/test/utils/spawn-child-process.js

```diff
@@ -0,0 +1,47 @@
+const { spawn } = require('child_process');
+
```

Comment:
I would suggest creating `Utils` class that has those 2 functions and the whole class is exported. It is the better `ES6` approach compared to `ES5` (old) approach